### DISCUSSION TITLE
Alternatives for service's dependencies

### DISCUSSION THREAD

**Discussion by realroot2185** [davmac314/dinit#339] [Ideas]
I looked a few display manager dinit services, from Artix, and they have:
`depends-on      = logind`

Something that allows another service as dependency could be useful:
`depends-on      = logind | seatd`

As far as I know there is no such feature.

**Comment by capezotte** [comment]
This can be done by making `logind` an internal service that, depending on configuration, pulls either `seatd` or some other implementation.

In fact, this is what Artix does: see the `logind` file of the [seatd](https://gitea.artixlinux.org/packages/seatd-dinit) and [elogind](https://gitea.artixlinux.org/packages/elogind-dinit) packages.

**Reply by realroot2185** [reply]
I confused `logind` with `elogind`.

I guess that's why seatd-dinit and elogind-dinit conflict because they share the same file (`logind`).
Maybe is not ideal? Not sure if alternatives can help with this.
